### DISCUSSION TITLE
For student  benifits

### DISCUSSION THREAD

**Discussion by Robinsingh13** [community/community#133279] [GitHub Education]
### Hi everyone,

hii , how much time it take for benifits till now it not approved benefits it showing available soon

**Comment by ededuardomoreno** [comment]
🙋🏻‍♂️ same here

**Comment by soudblox** [comment]
same here

**Comment by tszdabee** [comment]
@Robinsingh13 @ededuardomoreno @soudblox It usually takes around 1-2 business days for the academic benefits of the GitHub Student Developer Pack to show up after being approved. If you still haven't received your benefits after approval within 5 days, please raise another discussion so it can be addressed.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)

**Comment by romhenri** [comment]
On my experience 3 business days.

**Comment by Almoshaks964** [comment]
ملف تعارف الارتباط

**Comment by Almoshaks964** [comment]
كالي لينكس
